tools/c7n_mailer/setup.py

```diff
@@ -26,7 +26,7 @@
     "python-dateutil>=2.8",
     "ruamel.yaml>=0.15.93",
     "datadog",
-    "sendgrid==6.0.5",
+    "sendgrid>=6.0.5,<7.0",
```

Comment:
Is there a reason you didn't just remove the pin?

Reply:
the seem to be doing semver and they broke compatibility with 6 which required code changes on our end, given that I'm basically whitelisting 6-7 to keep inline with their compatibility behavior. the deps we denote in a package really do need to be effectively perm, for installs to work, so for packages that don't have stable apis we need some form of pin.